be/src/storage/delta_column_group.cpp

```diff
@@ -198,8 +198,9 @@ Status DeltaColumnGroupListSerializer::_deserialize_delta_column_group_list(cons
 }
 
 void DeltaColumnGroupListHelper::garbage_collection(DeltaColumnGroupList& dcg_list, const TabletSegmentId& tsid,
-                                                    int64_t min_readable_version,
-                                                    std::vector<std::pair<TabletSegmentId, int64_t>>& garbage_dcgs) {
+                                                    int64_t min_readable_version, const std::string& tablet_path,
+                                                    std::vector<std::pair<TabletSegmentId, int64_t>>& garbage_dcgs,
```

Comment:
Use `std::vector<std::pair<TabletSegmentId, int64_t>>*` when you want to change the parameter. Reference is better to be prefixed with `const`

Reply:
Done